### PR TITLE
FIX: Release tool image parsing handling '/' in tag names

### PR DESCRIPTION
**Pull request by alfrunes**

### CODE REVIEW

**Comment by alfrunes** [comment]
@mender-test-bot start pipeline

File: extra/release_tool.py [hunk 1]
```diff
@@ -2895,8 +2895,8 @@ def do_integration_versions_including(args):
 
 
 def find_repo_path(name, paths):
-    """ Try to find the git repo 'name' under some known paths.
-        Return abspath or None if not found.
+    """Try to find the git repo 'name' under some known paths.
+    Return abspath or None if not found.
```
Comment (lluiscampos) [inline extra/release_tool.py:2899]: Here and below: these are unrelated changes and would be better to have them on a separate commit (facilitates cherry-picking)-

**Review by lluiscampos — APPROVED** [review]

**Review by tranchitella — APPROVED** [review]

**Comment by lluiscampos** [comment]
Thanks for the fix!

I'll merge myself as this is blocking other pipelines.
